src/Microsoft.AspNetCore.ResponseCaching/ResponseCache/DistributedResponseCache.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace Microsoft.AspNetCore.ResponseCaching
```

Comment:
I'll move all the storage related types to the internal namespace and make them internal types since they shouldn't be needed outside of response caching.